src/frontend/store/hooks/use-window-size.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from 'react';
+
+const breakpoints = {
+	small: 600,
```

Comment:
I'm on the fence about whether this should be a larger size (the next core size would be 782), but that doesn't have to block anything.